TFNetworkLayer.py

```diff
@@ -4127,8 +4128,10 @@ def __init__(self, n_out, filter_size, padding, strides=1, dilation_rate=1,
       input_data = input_data.copy_add_feature_dim()
     if self.output.is_batch_feature_major:
       input_data = input_data.copy_as_batch_feature_major()
+      x = check_input_dim(input_data.get_placeholder_as_batch_major(), 1, self.input_data.dim)
     else:
       input_data = input_data.copy_with_feature_dim_axis(-1)
+      x = check_input_dim(input_data.get_placeholder_as_batch_major(), -1, self.input_data.dim)
```

Comment:
Please remove that. That's both incorrect, and also does not belong into this pull request.

Reply:
Done